Documentation/cmdref/cilium-agent.md

```diff
@@ -114,6 +114,7 @@ cilium-agent [flags]
       --enable-k8s-api-discovery                                Enable discovery of Kubernetes API groups and resources with the discovery API
       --enable-k8s-endpoint-slice                               Enables k8s EndpointSlice feature in Cilium if the k8s cluster supports it (default true)
       --enable-k8s-event-handover                               Enable k8s event handover to kvstore for improved scalability
+      --enable-k8s-networkpolicy                                Enable k8s NetworkPolicy watch in the cilium agent
```

Comment:
I think this doesn't match the current description.

I would also consider improving the language a bit, e.g. `Enables K8s NetworkPolicy support`, since the meaning of `watch` might not be clear to users